datalad_next/create_sibling_webdav.py

```diff
@@ -481,8 +484,6 @@ def _create_storage_sibling(ds, url, name, existing, credential, export):
     )
 
 
-# TODO in create-sibling-ria something like this is spaghetti-coded, it should be
-# RF'ed eventually
```

Comment:
I'd prefer to keep this comment, or have an issue in -core about it.

Reply:
Leaving it in

Reply:
It seems you did not actually leave it in. I have filed https://github.com/datalad/datalad/issues/6644 now.